src/main/java/io/github/jhipster/domain/BaseEntity.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2016-2018 the original author or authors from the JHipster project.
+ *
+ * This file is part of the JHipster project, see http://www.jhipster.tech/
+ * for more information.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.jhipster.domain;
+
+import com.google.common.base.MoreObjects;
```

Comment:
Where does this come from?

Reply:
That's for the `toString`. See comment below.

Reply:
Yes I got that, my question was do we have a dependency already for this? i dont remember using something from google thats why?

Reply:
It's IntelliJ's fault :-) It uses it to generate the `toString`. And I did it so much on my project that I thought it was coming from JHipster.